### ISSUE TITLE
Signup email field initial value disregarded

### ISSUE THREAD

**Issue by pennersr** [pennersr/django-allauth#1737]
In case `email` is not set, any existing `initial` value becomes unset:

```
    def get_context_data(self, **kwargs):
        ...
        email = self.request.session.get('account_verified_email')
        ...
        for email_key in email_keys:
            form.fields[email_key].initial = email

```

**Comment by pennersr** [comment]
Fixed in 625be21b66d01bb6a6a4b82159cc40a03f326644

**Comment by pennersr** [comment]
https://codeberg.org/allauth/django-allauth/issues/1737
